test/unit/osgb/outmodel/AddressTest.scala

```diff
@@ -25,20 +26,20 @@ class AddressTest extends AnyFunSuite {
   test(
     """Given an address with only one line and a town
        then 'printable' with newline should generate the correct string""") {
-    val a = Address(List("ATown"), "some-town", "FX1 1XX", Some("GB-ENG"), UK)
+    val a = Address(List("ATown"), "some-town", "FX1 1XX", Some(England), UK)
```

Comment:
Have we changed the expected input?  Will this change break people sending in their current input?